### ISSUE TITLE
GitHub History

### ISSUE THREAD

**Issue by StuffnThingz** [Montana-Media-Arts/441-web-technologies#19]
I was up until 4am last night trying to get my site to push through on GitHub, but it kept telling me a had a file that was too large. I know I have to clear the history somehow, but I'm not sure how to do it. Anyone experience this before? Thanks!
-Toby

**Comment by fbrown222** [comment]
Yes I had that problem before! I was able to fix it, by hitting the "undo" button on the GitHub desktop. Its located in the bottom left corner under where you click "commit". BUT!! Be careful! It does revert your Acorn code back to what is was when you hit undo. I would suggest copying your files to notes before hitting undo, and that way you can just paste them back in after. Make sure you either replace your file with something smaller or delete it. i hope this helps and works for you!

**Comment by StuffnThingz** [comment]
@fbrown222  Thanks! I should have figured such a simple problem would keep me up until 4am lol!

**Comment by fbrown222** [comment]
No worries, @StuffnThingz , it took me a really long time to figure it out too! I was ripping my hair out haha
Wish I could've helped you sooner!
